examples/todoapp/lib/feature_statistics/views/statistics_page.dart

```diff
@@ -21,7 +20,7 @@ class StatsPage extends StatelessWidget {
         appBar: AppBar(
           title: Text(context.l10n.todos),
           actions: const [
-            AppTodoListBulkEditPopupMenuButtonWithDependencies(),
+            AppTodoListBulkEditPopupMenuButton(),
```

Comment:
Don't we need the dependencies here ?

Reply:
From what I've seen, the `AppTodoListBulkEditPopupMenuButtonWithDependencies` instantiates it's own dependencies. It looks up the wrong dependencies for the tests: instead of the mocks, it looks up real ones preventing tests from running successfully. 
I may be wrong, but may @nikolay-vasilev-prime correct me.

Reply:
@DDavidPrime  is correct! Using `*WithDependencies` makes the tests to fail, because we cannot inject the mocked dependencies.

Reply:
Here can be found more information - https://github.com/Prime-Holding/rx_bloc/issues/700